packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -277,6 +283,8 @@ interface IRuntimeMessageMetadata {
 const runGCKey = "FluidRunGC";
 // Local storage key to turn GC test mode on / off.
 const gcTestModeKey = "FluidGCTestMode";
+// Local storage key to turn GC sweep on / off.
+const runSweep = "FluidRunSweep";
```

Comment:
#nit maybe this also needs to be consistent with the *Key suffix

Reply:
yeah, it should, thanks :-).